opentelemetry-collector/molecule/containerized/converge.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Converge
+  hosts: all
+  tasks:
+    - name: "Include caos.opentelemetry_collector"
+      include_role:
+        name: "caos.opentelemetry_collector"
```

Comment:
```suggestion
        name: "opentelemetry-collector"
```

Reply:
Role names cannot have `-`. I used this namespace to move forward, we can rename it later.(Not sure if it will work w/o it)

```
Role names are limited to lowercase word characters (i.e., a-z, 0-9) and ‘_’. No special characters are allowed, including ‘.’, ‘-‘, and space. During import, any ‘.’ and ‘-‘ characters contained in the repository name or role_name will be replaced with ‘_’.
```

https://galaxy.ansible.com/docs/contributing/creating_role.html#role-names